libs/sql-schema-describer/src/mysql.rs

```diff
@@ -189,8 +189,7 @@ async fn get_all_columns(
                 table_name table_name
             FROM information_schema.columns
             WHERE table_schema = ?
-            ORDER BY column_name
-            COLLATE utf8_general_ci
+            ORDER BY ordinal_position
```

Comment:
Why was the `COLLATE` there before? Why don't we need it anymore?

Reply:
I guess the answer to the second question is that we are ordering by a numeric column now :-)

Reply:
The sorting rules were necessary because before string values were compared (`ORDER BY column_name`).